reference/toolstacks/php/drupal/customizing-settings-php.md

```diff
@@ -1,12 +1,13 @@
 # Customizing settings.php
 
-Platform.sh allows you to use your own `settings.php` that you commit to
-your Git repository.
+For applications using the `drupal` build flavor (those based on our [Drupal 7
+example](https://github.com/platformsh/platformsh-example-drupal/tree/7.x)),
+Platform.sh automatically generates a `settings.php` and `settings.local.php`
```

Comment:
It generates both? Or only the former if it's missing? That's different than the text being replaced.